woocommerce/woocommerce
# Do not allow selecting text in the Paginator component of the Product editor

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes https://github.com/woocommerce/woocommerce/issues/40501



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Ensure to enable the New Product Feature ( WooCommerce -> Advances)
2. Before checking this branch (trunk)
  a. Go to the Product Editor -> Variations
  b. Ensure to add some variations
  c. Confirm if you click again and again in the variations paginator, it selects text (#40501)
3. Applying this PR changes
  a. Go to the Product Editor -> Variations
  b. Confirm it isn't possible to select text when clicking on the paginator


https://github.com/woocommerce/woocommerce/assets/77539/55351b02-4556-4e59-805f-e5a8ddbd31fe



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
